tests/test_cli.py

```diff
@@ -219,7 +356,7 @@ def test_script_main():
     assert sorted(data_out.files) == sorted(data_out.files) == sorted(
         ['embedding', 'timestamps'])
     assert np.allclose(data_out['timestamps'], data_reg['timestamps'],
-                       rtol=1e-05, atol=1e-06, equal_nan=False)
+                       rtol=1e-05, atol=1e-05, equal_nan=False)
     assert np.allclose(data_out['embedding'], data_reg['embedding'],
-                       rtol=1e-05, atol=1e-06, equal_nan=False)
+                       rtol=1e-05, atol=1e-05, equal_nan=False)
 
```

Comment:
Generally are we missing tests for video files for the CLI?

Reply:
No, every instance of "bento" refers to a video